src/Avalonia.FreeDesktop/DBusHelper.cs

```diff
@@ -56,19 +17,14 @@ public void Initialized()
             var oldContext = SynchronizationContext.Current;
             try
             {
-
-                var dbusContext = new DBusSyncContext();
-                SynchronizationContext.SetSynchronizationContext(dbusContext);
-                var conn = new Connection(new ClientConnectionOptions(dbusAddress ?? Address.Session)
+                var conn = new Connection(new ClientConnectionOptions(dbusAddress ?? Address.Session!)
                 {
-                    AutoConnect = false,
-                    SynchronizationContext = dbusContext
+                    AutoConnect = false
                 });
+
                 // Connect synchronously
-                conn.ConnectAsync().Wait();
+                conn.ConnectAsync().GetAwaiter().GetResult();
```

Comment:
Can we make TryInitialize async perhaps? To avoid this code smell?

Reply:
That'd either require the usage of an async void method (which, at least in my opinion, isn't any better), or making the complete initialization process async as well. While I'd like to see the latter at some point since nearly everything is async today anyway, it's probably out of scope for this PR.

Reply:
Ahh cool cool, the latter is a good thing to do afterwards 👍